src/memtest.c

```diff
@@ -347,10 +347,12 @@ void memtest_alloc_and_test(size_t megabytes, int passes) {
 }
 
 void memtest(size_t megabytes, int passes) {
+#if !defined(__HAIKU__)
     if (ioctl(1, TIOCGWINSZ, &ws) == -1) {
         ws.ws_col = 80;
         ws.ws_row = 20;
     }
+#endif
```

Comment:
@devnexen the defaults (80x25) still need to be set somehow. maybe at the top when declaring that function.
i assumed you ifdefed this since TIOCGWINSZ doesn't exist?

Reply:
Yes good point.